lib/teslamate/vehicles/vehicle.ex

```diff
@@ -394,7 +394,7 @@ defmodule TeslaMate.Vehicles.Vehicle do
 
     case stream_data do
       %Stream.Data{} when stale_stream_data? ->
-        Logger.warn("Received stale stream data: #{inspect(stream_data)}", car_id: data.car.id)
```

Comment:
This `inspect` call was essential to debug an issue with the stale streaming responses we had experienced in the past. It should not be removed IMO.